tools/pipelines/templates/build-npm-package.yml

```diff
@@ -383,7 +389,7 @@ stages:
             env:
               ADO_API_TOKEN: $(System.AccessToken)
               DANGER_GITHUB_API_TOKEN: $(githubPublicRepoSecret)
-              TARGET_BRANCH_NAME: $(System.PullRequest.TargetBranch)
+              TARGET_BRANCH_NAME: variables['targetBranchName']
```

Comment:
Did we have to use this syntax for a reason, instead of something like `${{ variables.targetBranchName }}`? I'd rather see a consistent syntax for this kind of access unless strictly necessary.

Reply:
Updated. I saw the format `variables[xxx]` is used in conditions checking, not sure if they need to be changed as well  https://github.com/microsoft/FluidFramework/blob/21b171b620ec56024eb16bc2ab0c6110a0f6daef/tools/pipelines/templates/build-npm-package.yml#L240

Reply:
Let's leave them like that where they're already used. I wouldn't be opposed to changing them but it's not a huge deal either way so better to not add unrelated changes to this PR.